Tests/BasicTests/DictionaryLiteralExtensionsTests.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+import Basic
+
+class DictionaryLiteralExtensionsTests: XCTestCase {
```

Comment:
What a shame!
[Addressed](https://github.com/apple/swift-package-manager/pull/1308/files#diff-5c66c87cc7ab455c0bb756d2b9d2278cR25) and squashed commits to one just in case 🙇